packages/cli/src/lib/pg/backups.ts

```diff
@@ -49,53 +54,147 @@ function prefix(transfer: BackupTransfer) {
   }
 }
 
-function filesize(size: number, opts = {}): string {
-  Object.assign(opts, {
-    decimalPlaces: 2,
-    fixedDecimals: true,
-  })
-  return bytes(size, opts)
-}
+class Backups {
+  public transfer: Pick<Backups, 'status' | 'num' | 'name'> // legacy purposes only
```

Comment:
what does the legacy comment mean exactly?

Reply:
This was mostly to avoid refactoring areas that still used `transfer.(status | num | name)` but it seems like leaving this would cause more confusion than just refactoring. I'll refactor!

Reply:
refactored!